MetadataProcessor.Shared/Tables/nanoSignaturesTable.cs

```diff
@@ -727,6 +840,21 @@ private void WriteAttributeArgumentValue(
             }
         }
 
+        private void WriteGenericParameterValue(BinaryWriter writer, GenericParameter parameter)
+        {
+            if (_context.GenericParamsTable.TryGetParameterId(parameter, out ushort referenceId))
+            {
+                writer.Write((byte)nanoSerializationType.ELEMENT_TYPE_GENERICINST);
+
+                // OK to use byte because we are not supporting more than 0x7F generic parameters
+                writer.Write((byte)referenceId);
+            }
+            else
+            {
+                // TODO
```

Comment:
maybe describe the work to be done and why it's ok not to do it now?

Reply:
Nice catch. Added a debug fail.